ui/jquery.ui.droppable.js

```diff
@@ -54,9 +55,15 @@ $.widget("ui.droppable", {
 		//Store the droppable's proportions
 		this.proportions = { width: this.element[0].offsetWidth, height: this.element[0].offsetHeight };
 
+		//Store the droppable's depth in the DOM tree
+		this.depth = 0;
+		while ( tgt.parent().size() >= 1 ) {
+			++this.depth;
+			tgt = tgt.parent();
```

Comment:
Why not just use the length for the depth and grab the last element? This loop isn't necessary.